charts/ingress-nginx/README.md

```diff
@@ -428,6 +428,7 @@ As of version `1.26.0` of this chart, by simply not providing any clusterIP valu
 | controller.service.internal.loadBalancerSourceRanges | list | `[]` | Restrict access For LoadBalancer service. Defaults to 0.0.0.0/0. |
 | controller.service.internal.ports | object | `{}` | Custom port mapping for internal service |
 | controller.service.internal.targetPorts | object | `{}` | Custom target port mapping for internal service |
+| controller.service.internal.type | string | `"LoadBalancer"` | Addes the abilaty to have a diffrnet service type for the internal load balancer |
```

Comment:
Re-run `helm-docs` after changing the `values.yaml` part according to my suggestions.